geospaas_harvesting/ingesters.py

```diff
@@ -510,6 +513,67 @@ def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
         return normalized_attributes
 
 
+class NetCDFIngester(MetanormIngester):
+    """Ingests metadata from NetCDF files. The files can be either
+    local or remote (if the remote repository supports it).
+    """
+
+    def _get_geometry_wkt(self, dataset):
+        """Extracts the WKT string for the dataset's spatial coverage
+        """
+        raise NotImplementedError()
+
+    def _get_raw_attributes(self, dataset_path):
+        """Get the raw metadata from the NetCDF file"""
+        dataset = netCDF4.Dataset(dataset_path)
+        raw_attributes = dataset.__dict__
```

Comment:
I opened a jaxa file with this dictionary:
![image](https://user-images.githubusercontent.com/62345618/107346829-7214cf00-6ac5-11eb-9b8c-40371ab67053.png)
how do you handle that?

Reply:
How do you handle what?

Reply:
I discussed it in below messages. To be honest, I first click on the review button and there is some discrepancy for my review comments here because I have to write them once again.